### PR TITLE
feat: color feature

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-4-4

### CODE REVIEW

File: note-app/src/Component/Main/Main.css [hunk 1]
```diff
@@ -37,5 +36,36 @@
...
+}
+.note-color{
+    width: 2rem;
+  height: 2rem;
+  border: 1px solid var(--secondary-color);
+  border-radius: 50%;
+  cursor: pointer;
+}
+.note-color-pink{
+    background-color: #f28b82;
```
Comment (shubhamkumar648) [inline note-app/src/Component/Main/Main.css:57]: here you can also use to put value as variable

**Review by shubhamkumar648 — APPROVED** [review]
code looks good and understandable, good use  use reducer hooks Great  UI design , good to merge

**Review by Rohitprasad83 — APPROVED** [review]
Code looks good to merge

File: note-app/src/Component/Main/Main.css (hunk 1 above)
Comment (aniketsingh13) [reply note-app/src/Component/Main/Main.css:57]: will do that

**Pull request merged**